### DISCUSSION TITLE
WCIU-DT5 (MeTV+)

### DISCUSSION THREAD

**Discussion by stanloonabelike** [iptv-org/community#789] [Q&A]
### Channel Name

METV+

### Channel ID

WCIUDT5.us

### Website

https://www.metv.com/metvplus

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by princo17** [comment]
le lien svp

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/blob/aec1bdb0a7037bf6f7112d2be3bdd93a50d5209f/streams/us.m3u#L695-L696
